ragna/deploy/_ui/css/modal_configuration/intslider.css

```diff
@@ -4,7 +4,7 @@
 }
 
 :host(.modal_configuration_int_slider) {
-  margin-top: 20px;
+  margin-top: 0px;
```

Comment:
As there are many more fields, reducing white space to try and fit everything into one modal.